.travis/script.sh

```diff
@@ -6,8 +6,8 @@ coverage erase
 python setup.py clean
 CFLAGS="-coverage" python setup.py build_ext --inplace
 
-coverage run --append --include=PIL/* selftest.py
-coverage run --append --include=PIL/* -m nose -vx Tests/test_*.py
+python selftest.py
+python setup.py test
```

Comment:
Is this going to do coverage without having it in the defaults?

Reply:
Good point, there's no coverage on Travis like this. I've reverted this commit.

Reply:
(previous comment deleted)

Updated to not use `python setup.py test`, and therefore not use setup.cfg's defaults.